docs/01_nodeos/03_plugins/chain_plugin/index.md

```diff
@@ -113,6 +113,18 @@ Config Options for eosio::chain_plugin:
                                         Safely shut down node when free space 
                                         remaining in the chain state database 
                                         drops below this size (in MiB).
+  --backing-store arg (=chainbase)      The storage for state, chainbase or 
+                                        rocksdb
+  --rocksdb-threads arg (=1)            Number of rocksdb threads for flush and
+                                        compaction
+  --rocksdb-files arg (=-1)             Max number of rocksdb files to keep 
+                                        open. -1 = unlimited.
+  --rocksdb-write-buffer-size arg (=134217728)
```

Comment:
Changed to rocksdb-write-buffer-size-mb

Reply:
Updated name to "rocksdb-write-buffer-size-mb"